src/components/essences/essences-toolbar.tsx

```
@@ -1,97 +1,145 @@
+                { startTransition: searchStartTransition },
+              );
+            }
+          }}
+        />
+        {searchPending && <span className="loading loading-xs shrink-0" />}
+      </label>
+
+      <div className="dropdown">
+        <summary
+          className="input w-full cursor-pointer select-none sm:w-auto"
+          role="button"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)